Customization/T200/CodeSnippets/Step2.1.2/RSSVDevice.cs.bak

```diff
@@ -0,0 +1,116 @@
+﻿using System;
```

Comment:
A bak file should not be here.

Reply:
Removed the file and updated files in the PhoneRepairProject_Code folder